src/components/atoms/masthead/components/Masthead.spec.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Masthead } from './Masthead';
+import { render } from 'react-testing-library';
+
+const imgSrc = 'https://peepo.gg/';
+const defaultProps = {
+    src: imgSrc
+};
+
+describe('Masthead component', () => {
+    const renderer = props => render(<Masthead { ...props } />);
```

Comment:
I know it's being picky, but I think this should be called `renderComponent` instead